Sources/TuistSupport/Utils/FileHandler.swift

```diff
@@ -254,4 +271,10 @@ public class FileHandler: FileHandling {
             return locateDirectoryTraversingParents(from: from.parentDirectory, path: path)
         }
     }
+
+    private func log(function: StaticString = #function, _ arguments: CustomStringConvertible...) {
+        if verbose {
+            Printer.shared.print("📂 \(function)\n\t\(arguments.map(\.description).joined(separator: " "))")
```

Comment:
Don't you think using the function name might be too verbose? I'd humanise a bit because method names are implementation details that the user doesn't care about. I'd expect a format along the lines of:

`Copying file from path x to path y`
`Traversing the paths from x to find a file that matches the path y`

And I'd remove the emoji because it might cause some issues when used on CI environments.

Reply:
I will remove this function and just use `Printer` directly and add more verbosity to the logs.